app/assets/javascripts/modules/track-links.js

```diff
@@ -5,7 +5,7 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
   'use strict'
 
   Modules.TrackLinks = function () {
-    this.start = function (element) {
+    this.init = function (element) {
```

Comment:
I'm not certain, but I think changing this to use `init` instead of `start` involves slightly more than renaming the function. If you take a look at https://github.com/alphagov/govuk_publishing_components/blob/master/app/assets/javascripts/govuk_publishing_components/components/details.js#L8 you'll see that this component receives the element it acts on (`$module`) in the initial function, which is then set as a variable `this.$module`, which is then accessed directly from the `init` function rather than being passed to it. There's also a slight difference at the end of the file in how the whole thing gets added to the `window.GOVUK.Modules` object.

I'm basing this on https://github.com/alphagov/govuk_publishing_components/pull/2095. @alex-ju will be able to provide more information about why it should be like this, but I think he's not in today.

Reply:
Kevin covered the difference between the `start` API and the `init` API, but it seems like you've already addressed this.